scenarios/csharp/GitHubLovesSparkDotNet/README.md

```diff
@@ -0,0 +1,77 @@
+In the **Github 💖 .NET for Apache Spark** app, we dive into the world of
+analyzing GitHub meta-data using .NET for Apache Spark. While the analysis
+is simple, it conveys a key point of how easy it is to get started with
+analyzing large datasets using .NET.
+
+# Table of Contents
+
+- [Introduction](#introduction)
+- [Technology Stacks](#technology-stacks)
+- [Pre-requisites](#pre-requisites)
+- [Running the App](#running-the-app)
+- [Discussion](#discussion)
+- [Insights](#insights)
```

Comment:
Remove ?  I don't see an insights section below.

Reply:
I'll add this now. Thank you!